src/stservo/gui/servo_gui.py

```
@@ -1544,6 +1701,280 @@ def clear_sync_results(self):
+            self.register_results_text.insert(tk.END, "="*80 + "\n\n")
+            
+            for addr in sorted(self.registers.keys()):
+                reg_info = self.registers[addr]
+                
+                try:
+                    if reg_info['size'] == 1:
+                        value, comm_result, error = self.packet_handler.read1ByteTxRx(self.current_servo_id, addr)
+                    else:
+                        value, comm_result, error = self.packet_handler.read2ByteTxRx(self.current_servo_id, addr)
+                    
+                    if comm_result == COMM_SUCCESS and error == 0:
```

Ruff (0.12.2)

`COMM_SUCCESS` may be undefined, or defined from star imports

(F405)